cypress/integration/login/login_connect_orcid.js

```diff
@@ -0,0 +1,36 @@
+import { Given, And, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { USER_CONNECT_ORCID, NAME_CONNECT_ORCID } from '../../support/constants';
+import { createUser, addQualifierToAuthority } from '../../support/users';
+
+Given('that the user has valid Feide credentials', () => {});
+And('they have logged in with Feide before', () => {
+  createUser(USER_CONNECT_ORCID, NAME_CONNECT_ORCID, true, USER_CONNECT_ORCID).then((authority) => {
+    cy.wrap(authority).as('authority');
+    cy.get('@idToken').then((idToken) => {
+      cy.addQualifierId(authority.systemControlNumber, 'feide', USER_CONNECT_ORCID, idToken).then(() => {
+        cy.addQualifierId(authority.systemControlNumber, 'orgunitid', '2020.0.0.0', idToken);
+      });
+    });
+  });
+});
+And('they have not connected ORCID', () => {});
+And('they are on the Start page', () => {
+  cy.get('@authority').then((authority) => {
+    cy.visit('/');
+  });
+});
+When('they click Log in', () => {});

```

Comment:
burde ikke funksjonen her trykke på "Log in"-knappen?

Reply:
Log in vil sende deg til Feide og det fungerer ikke med Cypress